charts/seed-monitoring/charts/grafana/templates/grafana-deployment.yaml

```diff
@@ -29,10 +29,6 @@ spec:
         role: {{ .Values.role }}
         networking.gardener.cloud/to-loki: allowed
     spec:
-      initContainers:
-      - name: init-prometheus
-        image: {{ index .Values.images "busybox" }}
-        command: ['sh', '-c', 'until wget -T 5 -qO- http://prometheus-web/-/healthy > /dev/null; do echo waiting for Prometheus; sleep 2; done;']
```

Comment:
I don't know the full details about this dependency between grafana and prometheus, but understand that without removing it you still cannot access the logs from the loki as grafana simply cannot start-up.

It will be nice to have some feedback from @wyb1 and @istvanballok on this change. 

But while you are decoupling grafana from prometheus, please reduce the `failureThreshold` of the readiness and liveness probes of prometheus here https://github.com/gardener/gardener/blob/19f46386468b601f3fb9edea74c0cd893d788c53/charts/seed-monitoring/charts/core/charts/prometheus/templates/prometheus.yaml#L87-L104

I would expect this threshold to have value between 1 and 3 usually, not 60 and 120.

Reply:
I guess you are right for the `livenessProbe` but for `readinessProbe`, I would not agree.
If we like to give more time to Prometheuse to bring up we should increase the time of the `readinessProbe` before it fails. On the other hand, we wish to mark it as ready as soon as possible so we don't want to make a probe every 200 seconds.

Reply:
`failureThreshold` is not measured in time but in occurrences. Currently the readiness probe has to fail 120 times in a row with 5 seconds interval between each probe to finally consider the pod as not ready. In result, the pod may receive traffic for 10 full minutes when it is actually not able to accept it. And the liveness probe having `failureThreshold=60` will most probably restart the container before even to k8s to consider the pod as not ready.

As prometheus is starting slowly, it can use `initialDelaySeconds` to delay the probes.

Now I think it was not good idea to ask you to change it, probably it deserves own PR.

Reply:
Without Prometheus all of the grafana dashboards that use Prometheus as a datasource would be empty. This is why it was originally introduced since grafana is pretty useless without the data, so we wait for prometheus. Since there are some other use cases it can be safely removed. If Prometheus is not up the only consequence is that the dashboards using Prometheus as a datasource will be empty.